lib/utils.ts

```diff
@@ -87,3 +89,7 @@ export const verifyClaim = (token: string, { iss }: ITokenPayload) => {
   const addressFromDigest = recoverAddress(digest, signature)
   return decodedAddress === addressFromDigest ? iss : ''
 }
+
+export const decodeJwToken = (token: string) : any => {
```

Comment:
Is there a `@types` package for `jsonwebtoken` so that the specific type can be specified instead of returning `any`?

Reply:
Yes ok, I wiil fix it. The returned type is `string | { [key: string]: any; }`